apps/logsupport.cpp

```diff
@@ -95,6 +95,8 @@ set<srt_logging::LogFA> SrtParseLogFA(string fa, set<string>* punknown)
         fas.insert(SRT_LOGFA_DATA);
         fas.insert(SRT_LOGFA_TSBPD);
         fas.insert(SRT_LOGFA_REXMIT);
+        fas.insert(SRT_LOGFA_HAICRYPT);
```

Comment:
Haicrypt shouldn't be among those turned on by default, as several information displayed from there may be unsafe.

Reply:
Removed.